viz/README.md

```diff
@@ -54,8 +54,6 @@ You are ready to launch the visualization now.
 Note: If you have created the JSON file externally then simply copy it over in the `viz/data` subdirectory and run the `geninputfile.py` script using the above command.
```

Comment:
This is a little bit misleading. You either need to copy `opflowout.json` to `viz` directory or `geninputfile.py` to `viz/data`. Once can figure it out, but the instructions should be precise.

Is it possible to do this in a separate directory outside ExaGO source tree? That would be preferable.